shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -888,16 +895,17 @@ - (instancetype)initWithTarget:(id)target
 }
 
 - (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
-  [_platformViewsController->getFlutterViewController() touchesBegan:touches withEvent:event];
+  _flutterViewController = _platformViewsController->getFlutterViewController();
```

Comment:
Is it possible that `_platformViewsController->getFlutterViewController()` could return different values between calls `touchesBegan` before `touchesEnded` has happened?

1) touch-1 began
1) flutter view controller changed
1) touch-2 began
1) touch-1 ended
1) touch-2 ended

If so you'd have to be super vigilant and track the FlutterViewController at touchesBegan for each event.

Reply:
Has this been addressed in the recent commit(s)?

Reply:
Updated the code to support this scenario. I also added a test for this.